packages/workflow/src/Interfaces.ts

```diff
@@ -1362,6 +1362,8 @@ export interface INodePropertyTypeOptions {
 	resourceMapper?: ResourceMapperTypeOptions;
 	filter?: FilterTypeOptions;
 	assignment?: AssignmentTypeOptions;
+	minFieldCount?: number; // Supported by: fixedCollection
```

Comment:
can we improve the naming 
```suggestion
	minRequiredFields?: number; // Supported by: fixedCollection
	maxAllowedFields?: number; // Supported by: fixedCollection
```

Reply:
Sure, nice suggestions